orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/restore.sls

```diff
@@ -0,0 +1,9 @@
+include:
+  - postgresql.disaster_recovery
+
+restore_postgresql_db:
+  cmd.run:
+    - name: /opt/salt/scripts/restore_db.sh {{salt['pillar.get']('platform')}} {{salt['pillar.get']('disaster_recovery:object_storage_url')}} {{salt['pillar.get']('postgres:remote_db_url')}} {{salt['pillar.get']('postgres:remote_db_port')}} {{salt['pillar.get']('postgres:remote_admin')}}
```

Comment:
Not all of our clusters use external DB. Many tests use embedded DB where these pillar configs will be missing. We should add a condition to only execute this if it's external DB. See: [example](https://github.com/hortonworks/cloudbreak/blob/master/orchestrator-salt/src/main/resources/salt/salt/postgresql/init.sls#L3-L5)

Reply:
Should we expect that customers will be running with clusters that are using embedded DBs? 
This may be a question for @kkalvagadda1 as well.

 **Are we performing DR with embedded DBs as well?**